src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/DynamoTableDescriptionImpl.java

```diff
@@ -115,10 +122,11 @@ public PrimaryKey getPrimaryKey() {
     }
 
     private ScalarAttributeType getAttributeType(String attributeName) {
-        return ScalarAttributeType.valueOf(attributeDefinitions.stream()
+        Set<AttributeDefinition> filteredAttributeDefinitions = this.attributeDefinitions.stream()
             .filter(attributeDefinition -> attributeDefinition.getAttributeName().equals(attributeName))
-            .findFirst().orElseThrow(() -> new IllegalArgumentException("attribute with name '" + attributeName
-                + "' not found in " + attributeDefinitions)).getAttributeType());
+            .collect(Collectors.toSet());
+        checkArgument(!filteredAttributeDefinitions.isEmpty());
```

Comment:
add message? (e.g., `"attribute with name '" + attributeName + "' not found in " + attributeDefinitions`)